tools/cli/help.txt

```diff
@@ -1,19 +1,25 @@
 >>>
 (the first line of each entry is the summary for autogenerated command lists)
-Usage: meteor [--release <release>] [--help] <command> [args]
-       meteor help <command>
+Usage: meteor [--release <release>] [--help] <built-in command> [args]
+       meteor help <built-in command>
+       meteor <external command> [args]       
```

Comment:
Hmm, an "npm command" to me, is `install`, `ls`, `version`, `shrinkwrap`, etc., so not sure if I agree with "npm command".  "Third-party command"?  "Add-on command"?